utility/module.go

```diff
@@ -21,14 +23,31 @@ const (
 )
 
 func Create(configPath, genesisPath string) (modules.UtilityModule, error) {
+	u := new(UtilityModule)
+	c, err := u.InitConfig(configPath)
+	if err != nil {
+		return nil, err
+	}
+	config := (c).(*types.UtilityConfig)
 	return &UtilityModule{
-		// TODO: Add `maxTransactionBytes` and `maxTransactions` to cfg.Utility
-		Mempool: types.NewMempool(1000, 1000),
+		Mempool: types.NewMempool(config.Max_Mempool_Transaction_Bytes, config.Max_Mempool_Transactions),
 	}, nil
 }
 
 func (u *UtilityModule) InitConfig(pathToConfigJSON string) (config modules.IConfig, err error) {
```

Comment:
I'm thinking we should aim to review & merge in #235 first and rebase this on top of the new pattern.

Reply:
Why is that prioritized over this small one?

Reply:
Alright, let's prioritize this one first.